src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -2342,6 +2342,21 @@ public void TestMillisConversion()
             }
         }
 
+        [Fact]
+        public void TestSelectEmptyClause()
+        {
+            int resultCnt = 0;
+            string k1 = "";
+            string k2 = "";
+            LoadNumbers(100);
+            using (var q = QueryBuilder.Select().From(DataSource.Database(Db))
+                .Where(Expression.Property("number1").LessThan(Expression.Int(10))).OrderBy(Ordering.Property("number1"))) {
+                var res = q.Execute();
+                resultCnt = res.Count();
+            }
+            resultCnt.Should().Be(9);

```

Comment:
The test is for the naming of the column so please check the name of the column and make sure it is correct.